aclk/agent_cloud_link.h

```diff
@@ -6,20 +6,21 @@
 #include "../daemon/common.h"
 #include "mqtt.h"
 
+#define ACLK_VERSION 1
+#define ACLK_THREAD_NAME "ACLKQ"
```

Comment:
as the main thread is called `ACLK_Main`. I propose this one should be called `ACLK_Query`

Reply:
Sure .. good idea